### PR TITLE
Updates to work with Featuretools 0.16.0

### PR DESCRIPTION
**Pull request by thehomebrewnerd**
Updated requirements to suppress bokeh and scikit-learn warnings.

Updated notebook cutoff time dataframe so that notebook still works with Featuretools version 0.16.0.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
-bokeh>=1.0.2
+bokeh>=1.0.2,<1.3.0
 featuretools>=0.6.0
```
Comment (rwedge) [inline requirements.txt]: can we bump this to 0.16

Comment (thehomebrewnerd) [reply requirements.txt]: We can, but we don't need to as this particular notebook will still work with older versions of Featuretools. Probably would be best to bump though, in light of the comment I added to the notebook regarding naming of cutoff time columns.

Comment (thehomebrewnerd) [reply requirements.txt]: Bumped

**Comment by rwedge** [comment]
Looks good

**Review by rwedge — APPROVED** [review]

**Pull request merged**